src/components/FilterableProductTable.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import SearchBar from './SearchBar';
+import ProductTable from './ProductTable';
+
+class FilterableProductTable extends React.Component {

```

Comment:
Your app is working properly, I just would suggest you with time, to make the searchbar more dinamically and make it update when you delete one letter and not only when you clean the search values. Nevertheless, it works as asked!